raiden/tests/unit/test_accounts.py

```diff
@@ -80,7 +80,7 @@ def test_account_manager_invalid_files(test_keystore, caplog):
         AccountManager(test_keystore)
 
     for msg, file_name, reason in [
-        ('Invalid account file', KEYFILE_INVALID, 'No JSON object'),
+        ('Invalid account file', KEYFILE_INVALID, 'Expecting value: line 1 column 1 (char 0)'),
```

Comment:
This feels so hacky ...

Reply:
Does it? It was just the error message of the exception that was nicer before. We should handle that better in `AccountManager` as Augusto said.

Reply:
See #1195